### PR TITLE
refactor: added animation and favicon

### PR DESCRIPTION
**Pull request by boby-boo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>SKYDRONE</title>
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+    <link rel="shortcut icon" href="/src/favicon.ico">
```
Comment (PKulkoRaccoonGang) [inline index.html:9]: Do we need this slash?
```suggestion
    <link rel="shortcut icon" href="src/favicon.ico">
```

File: styles/services.css [hunk 2]
```diff
@@ -9,8 +10,17 @@
...
 
+@keyframes servicesRow {
+    0% {
+        transform: translateX(100%);
+    }
+    100% {
+        transform: translateX(-100%);
+    }
+}
 .services-row li {
```
Comment (PKulkoRaccoonGang) [inline styles/services.css:24]: Please, add an empty line after `@keyframes {} `declaration

File: index.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
...
     <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/style.css">
     <link rel="stylesheet" href="./styles/header.css">
```
Comment (PKulkoRaccoonGang) [inline index.html:12]: Do we need this `./` ?
```suggestion
    <link rel="stylesheet" href="styles/header.css">
```

File: styles/services.css (hunk 2 above)
Comment (boby-boo) [reply styles/services.css:24]: fix

File: index.html (hunk 3 above)
Comment (boby-boo) [reply index.html:12]: fix

**Pull request merged**